update-rust-version.sh

```diff
@@ -0,0 +1,5 @@
+VERSION=$1
```

Comment:
This seems like it should probably have a `#!/bin/bash` shebang?

Reply:
yes totally forgot it 😅